src/software/world/field.h

```diff
@@ -162,6 +162,13 @@ class Field
      */
     double defenseAreaXLength() const;
 
+    /**
+     * The distance from the goal centre to the penalty mark
+     *
+     * @return the x coordinate distance from the goal to the penalty mark
+     */
+    double goalToPenaltyMark() const;
```

Comment:
Can we modify penaltyFriendly/penaltyEnemy instead? (also rename to say penalty mark)

Reply:
renamed and also took akhil's suggestion from before